depends/config.site.in

```diff
@@ -14,7 +14,7 @@ if test -z $enable_wallet && test -n "@no_wallet@"; then
 fi
 
 RUST_TARGET="@rust_target@"
-RUST_VENDORED_SOURCES="$depends_prefix/vendored-sources"
+RUST_VENDORED_SOURCES="$depends_prefix/../sources/vendored-crates"
```

Comment:
@charlieok ran a Gitian build of this PR, and it failed because this path is only valid for the default value of `SOURCE_PATH`. Gitian builds override this to pre-cache the source files, and the PR correctly stores the vendored crates in that path. Fix is to substitute this path instead of hard-coding it.

Reply:
[Fixed!](https://github.com/zcash/zcash/compare/0381fb89ed6dfdd7f4d57f952c206e24aab1dddb..336559a9edb2d052d81e1be8184ed18255b883c4)